addons/test_mass_mailing/tests/test_mailing_statistics.py

```diff
@@ -53,6 +53,7 @@ def test_mailing_statistics(self):
         self.assertEqual(mailing.bounced, 1)
         self.assertEqual(mailing.bounced_ratio, 8.33)
         self.assertEqual(mailing.canceled, 1)
+        self.assertEqual(mailing.expected, 13)
```

Comment:
this change should be in the previous commit no ?

Reply:
@datonefaridze please read and update all comments, you mark as solved but it is not done. Don't try to be faster than light, in the end we lose time having to recheck the PR content to be sure everything is done as intended.

Reply:
@tde-banana-odoo i also checked this comment (asked my self the same question) and i thought std meant this comment (because first commit is about ui).

Reply:
@tde-banana-odoo Now i think i did everything correctly. (checked everything too, hope i am not making any mistakes)